corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -142,5 +142,34 @@ hqDefine("geospatial/js/gps_capture",[
 
     $(function () {
         $("#no-gps-list").koApplyBindings(dataItemListModel());
+
+        // Global var
+        var map;
+
+        var initMap = function (centerCoordinates) {
+            'use strict';
+
+            var self = {};
+            mapboxgl.accessToken = initialPageData.get('mapbox_access_token');
+
+            if (!centerCoordinates) {
+                centerCoordinates = [2.43333330, 9.750]; // should be domain specific
+            }
+
+            const map = new mapboxgl.Map({
```

Comment:
The `const` can be removed, since this variable has already been declared a few lines above. Currently, a new locally scoped map variable will be declared and `var map` will stay undefined.

Reply:
thanks @zandre-eng 

my understanding is that it is done to have "map" available outside the scope of this function?
Same as done [here](https://github.com/dimagi/commcare-hq/blob/e19fcc80f04e52d9540f96d613b4cdcba492751e/corehq/apps/geospatial/static/geospatial/js/geospatial_map.js#L47)

Reply:
In that example, `var map` and `const map` seem to be referencing different things. The var is referencing the `loadMapBox` class whilst the const is a reference to the actual MapBox map. The example code could use a bit of cleanup to better differentiate between the two, but for this PR the `const` can be removed since `var map` isn't being used to reference anything else.

Reply:
Thanks, I will check more about this.

Is `const` meant to restrict to have only one page per map? I am just guessing what the intention was when this was done in that example I linked.

Reply:
I got rid of the var map for now and updated the init function to return the map object.
That seemed reasonable for what the code does now. If needed for future, var can be put back for reference if needed.

Reply:
> Is const meant to restrict to have only one page per map? I am just guessing what the intention was when this was done in that example I linked.

That sounds right, I think the intention was to make sure we don't save a different map to that variable, since a `const` can only have its value set once. We can still technically add more maps to the page, however these will need to be assigned to different variables.